store/storetest/channel_store.go

```diff
@@ -7633,23 +7632,20 @@ func testChannelStoreGetChannelsBatchForIndexing(t *testing.T, ss store.Store) {
 	_, nErr = ss.Channel().Save(c6, -1)
 	require.NoError(t, nErr)
 
-	endTime := c6.CreateAt
-
 	// First and last channel should be outside the range
-	channels, err := ss.Channel().GetChannelsBatchForIndexing(startTime, endTime, 1000)
+	channels, err := ss.Channel().GetChannelsBatchForIndexing(c1.CreateAt, "", 4)
 	assert.NoError(t, err)
-	assert.ElementsMatch(t, []*model.Channel{c2, c3, c4, c5}, channels)
+	assert.Len(t, channels, 4)
```

Comment:
Shall we do an assertion like this to make sure we ignore the `c1` and `c6`? Maybe we can use this assertion on cases below?
```suggestion
	assert.NotContains(t, channels, []*model.Channel{c1, c6})
	assert.Len(t, channels, 4)
```

Reply:
With the new model, you cannot really guarantee exact order of elements. Because ids are random, so elements can be ordered randomly within the same timestamp. So we can only assert for number of elements, and verify that pagination works correctly till the end.